editor/editor.go

```diff
@@ -17,7 +17,14 @@ const defaultEditor = "nano"
 type Option func(editor, filename string) (args []string, pathInArgs bool)
 
 // OpenAtLine opens the file at the given line number in supported editors.
+//
+// Deprecated: use Line instead.
 func OpenAtLine(number uint) Option {
+	return Line(number)
+}
+
+// Line opens the file at the given line number in supported editors.
+func Line(number uint) Option {
```

Comment:
Merely a suggestion to make this option clearer 🤔

```suggestion
// LineNr is an option that opens the file at the given line number in supported editors.
func LineNr(number uint) Option {
```